security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc

```diff
@@ -77,6 +78,7 @@ function stunnel_save() {
 		fwrite($fout, base64_decode($pkgconfig['custom_options']) . "\n");
 	}
 	fclose($fout);
+	write_config();
```

Comment:
Please add a description to write_config()

Reply:
done